### DISCUSSION TITLE
Can we rename the plex watchlist made by trakt?

### DISCUSSION THREAD

**Discussion by hackmonker** [Taxel/PlexTraktSync#792] [Q&A] [answered]
I wanted to rename the playlist but will it still sync after renaming? if not can we specify the name of the playlist created maybe in the config?

**Comment by simonc56** [comment, marked as answer]
The answer is no for both questions.

**Comment by glensc** [comment]
if you rename the playlist, PTS will create new list with the trakt list name. the renamed playlist will remain unaltered by PTS.
